pymc/tests/test_distributions_random.py

```diff
@@ -417,28 +314,34 @@ def seeded_numpy_distribution_builder(dist_name: str) -> Callable:
     )
 
 
-class TestFlat(BaseTestDistributionRandom):
-    pymc_dist = pm.Flat
-    pymc_dist_params = {}
-    expected_rv_op_params = {}
+class TestGaussianRandomWalk(BaseTestDistributionRandom):
+    pymc_dist = pm.GaussianRandomWalk
+    pymc_dist_params = {"mu": 1.0, "steps": 2, "sigma": 3}
+    expected_rv_op_params = {"mu": 1.0, "sigma": 3, "steps": 2}
+    # reference_dist_params = {"b": 1.0, "kappa": 1.0, "mu": 0.0}
+
     checks_to_run = [
```

Comment:
Same here, I would move this to the test_distributions_timeseries file

Reply:
Agreed, Ill make an to move this in a future PR just so we can get this merged with minimal headache

Reply:
I don't think it makes sense to add new tests here if we already plan to move them.

For your other question, no need to test implied dims I think.

Reply:
I dont think we can move these because the test fixture is specified in this file, unless we move the whole fixture to conftest.py so its globally available, but I think thats a worse option

Reply:
I don't follow. You just need to import the test class from this file, no?